### PR TITLE
fix scripts to enable clean releases

### PR DESCRIPTION
**Pull request by lhein**
Signed-off-by: Lars Heinemann <lhein.smx@gmail.com>

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -12,19 +14,21 @@ env:
   global:
   - GPG_DIR="`pwd`/cd"
 after_success:
-- if [[ $TRAVIS_PULL_REQUEST == "false" && $TRAVIS_BRANCH == "master" ]]; then 
+- if [[ $IS_SNAPSHOT == "SNAPSHOT" ]]; then
```
Comment (apupier) [inline .travis.yml]: why IS_SNAPSHOT ans SNAPSHOT is used? what's the difference?

Comment (lhein) [reply .travis.yml]: it makes it easier for me as a non-bash-nerd to work with the variables. IS_SNAPSHOT contains "SNAPSHOT" if its a snapshot version while SNAPSHOT is set to TRUE or FALSE depending whats in IS_SNAPSHOT :D

Comment (apupier) [reply .travis.yml]: what do you think to change variable name to try to have it clearer?
For instance:
"IS_SNAPSHOT" to contain true/false
"VERSION_SUFFIX" to contain "SNAPSHOT"/*<no value>*

Comment (lhein) [reply .travis.yml]: I could do that but that will also require the changes on common.

**Pull request merged**